### ISSUE TITLE
The Write a review button is inaccessible from the keyboard

### ISSUE THREAD

**Issue by kumar303** [mozilla/addons-frontend#6579]
### Describe the problem and steps to reproduce it:

1. Sign in to AMO dev
2. Go to an add-on detail page
3. Navigate with the tab key on your keyboard until you reach the *Rate your experience* card
4. Select a star by using your keyboard
5. Try to press tab to move focus to the *Write a review* button

### What happened?

No matter how much you press the tab key, *Write a review* is never focused

### What did you expect to happen?

The *Write a review* button should be focused immediately.

### Anything else we should know?

After this is fixed, there might be another bug: https://github.com/mozilla/addons-frontend/issues/6246

**Comment by kumar303** [comment]
After this fix lands, we need to make sure everything works on Android. I need to change some things that may cause problems.

**Comment by AlexandraMoga** [comment]
Verified fixed on AMO -dev with FF63, Win10x64 and Android 8.0

I didn't find any regressions on Android around the ratings and review functionalities. 

![correct button selection](https://user-images.githubusercontent.com/31961530/48348599-7eade100-e68a-11e8-94b5-230b0849232f.gif)
